platform/pulpcore/app/viewsets/task.py

```diff
@@ -19,18 +20,19 @@ class Meta:
         fields = ['state', 'worker__name', 'tags']
 
 
-class TaskViewSet(NamedModelViewSet):
+class TaskViewSet(mixins.RetrieveModelMixin,
+                  mixins.ListModelMixin,
+                  GenericNamedModelViewSet):
     queryset = Task.objects.all()
     serializer_class = TaskSerializer
     endpoint_name = 'tasks'
     filter_class = TaskFilter
-    http_method_names = ['get', 'options']
```

Comment:
@goosemania, The `create` method was defined by the create mixin, which was inherited from the NamedModelViewSet, but POST itself was disallowed for every views that is a part of the TaskViewSet. I'm going to merge, but lets chat about this and make sure we are on the same page.